config/alliance_fee_share.json

```diff
@@ -30,5 +47,9 @@
       "partner_share_pct": 0.175,
       "dao_share_pct": 0.175
     }
+  },
+  "alliance_thresholds": {
+    "v3_min_tvl": 5000000,
+    "v2_min_tvl": 0
```

Comment:
added threshold values here. a value of 0 effectively means any pool added to this config is included